.devcontainer/basics/devcontainer.json

```diff
@@ -0,0 +1,43 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.238.1/containers/docker-existing-dockerfile
```

Comment:
> https://github.com/microsoft/vscode-dev-containers/tree/v0.238.1/containers/docker-existing-dockerfile

Wondering why would we need a reference to `docker-existing-dockerfile`? 🤔 
Also, the container in `microsoft/vscode-dev-containers` are not currently supported and are moved to the [dev containers](https://github.com/devcontainers) org.

Reply:
Probably copied from somewhere.